Samples/excel-create-worksheet-from-web-site/WebApplication/App/authConfig.js

```diff
@@ -8,8 +8,8 @@
  */
 const msalConfig = {
     auth: {
-        clientId: '9960ebe1-1aaf-4c22-8c3a-a769cbaf0422', // This is the ONLY mandatory field that you need to supply.
-        authority: 'https://login.microsoftonline.com/0c98caa3-55d0-48ba-a8ba-de76532669a2', // Defaults to "https://login.microsoftonline.com/common"
+        clientId: 'Enter_the_Application_Id_Here', // This is the ONLY mandatory field that you need to supply.
+        authority: 'https://login.microsoftonline.com/Enter_the_Tenant_Id_Here', // Defaults to "https://login.microsoftonline.com/common"
```

Comment:
Out of curiosity, what causes this to default to the /common URL?

Reply:
That's text from the original auth sample I used. I believe it is referring to how you configure the app registration. Multi-tenant support will use "common" while an actual tenant id restricts the usage to only that single tenant.